### ISSUE TITLE
Screen Reader Checklist is out of date following Zoom iOS updates

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#52281]
### Description
In a profile study conducted this morning, we encountered a user with an iPhone 13 and the newest version of zoom. The current guidance is out of date and does not accurately map to the button labels (e.g. "Share content" is now just "Share", and the order is different).

I found the outdated guidance (which I wrote in 2021) insufficient to support the study.

I'd recommend any of the following:
- updating the guidance to reflect current zoom settings (but this can lead into tricky situations where different versions may exist)
- creating a training program / general guidance or steps on prepping with an a11y specialist ahead of a study to practice with the participant's technology ahead of time
- building in explicit time to meet participants with assistive technology ahead of time to do tech checks (which may require needing to partner with Perigean) 
- addressing scenarios around sharing VO sound via microphone

### Relevant URLs
[Screen reader checklist](https://depo-platform-documentation.scrollhelp.site/research-design/screen-reader-checklist)

---

## User Story
As a VFS team member preparing for research, I want to be able to rely on and trust the Platform Website guidance so that I know I can complete my research without any challenges.

Assignee: @briandeconinck 
Peer Reviewer: @allison0034 

## Description
See Josh's description above.

## Impacted Artifacts
[Screen reader checklist](https://depo-platform-documentation.scrollhelp.site/research-design/screen-reader-checklist)

## Tasks

- [x] Verify Josh's recommended changes and identify any other updates that may be needed.
- [x] Update the screen reader checklist
   - [Draft screen reader checklist](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2528641047/52281+-+Screen+Reader+Checklist) (changes only made to the iOS VoiceOver section)

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] VFS team members are able to read and follow the Screen Reader Checklist guidance without encountering errors

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by vbellissimo** [comment]
Hi @joshkimux, thanks for your content suggestions. The Platform Content team will review your request and put it in our backlog to refine for next steps.

**Comment by shiragoodman** [comment]
@vbellissimo FYI I removed you as an assignee.

Governance team plans to pick this work up in Sprint 20 beginning 2/2.

**Comment by shiragoodman** [comment]
@vbellissimo can we remove the dependency on ticket #52281?

**Comment by vbellissimo** [comment]
@shiragoodman Yes, I removed the dependency and removed my team's tag, as well. Thank you.

**Comment by briandeconinck** [comment]
Status update: Scheduling a working session with Jasmine Friedrich this week to test and make sure I have the right steps.

**Comment by shiragoodman** [comment]
@briandeconinck could you please provide a status update on this ticket? I am aware the work is continuing into the next sprint, however I don't recall the specific update you shared today. ty!

**Comment by briandeconinck** [comment]
@allison0034 this is finally ready for review! Validated all of the changes in a call with Josh, and identified some new things that weren't previously accounted for. (Note that Josh's original ticket said "Share Content" had become "Share." That was an error.)

[Draft of Screen reader checklist](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2528641047/52281+-+Screen+Reader+Checklist) (all changes are in the iOS VoiceOver section)

**Comment by briandeconinck** [comment]
Changes made in Confluence, will close when published to Platform website tomorrow.

**Comment by briandeconinck** [comment]
Changes were published to Platform website yesterday: [Screen Reader Checklist](https://depo-platform-documentation.scrollhelp.site/research-design/screen-reader-checklist). Closing as complete!
